### ISSUE TITLE
Keyboard navigation stopped working after pausing auto play

### ISSUE THREAD

**Issue by nathandao** [tuunanen/browse-shit#6]
After trying out the Play button and got frustrated with the long interval between items, I decided to click Pause and holy crap - the keyboard navigation does not work any more. HUGE let off.

**Comment by tuunanen** [comment]
Sorry to hear you have had bad experience with the product.

This happens because the `keydown` event listener is bound to `BrowserControlsView` instead of the `<body>` element. When clicking the play/pause button, the focus is lost from the view element and consequently appropriate event callback is not invoked. Definite program design failure.
